cmd/plugin.go

```diff
@@ -306,6 +306,14 @@ func runPluginUpdateCmd(cmd *cobra.Command, args []string) {
 		return
 	}
 
+	if len(plugins) > 0 && !(cmdconfig.Viper().GetBool("all")) && plugins[0] == "all" {
```

Comment:
why do we need `!(cmdconfig.Viper().GetBool("all"))` in the if?

Reply:
to check if there's any `--all` flag